ethportal-api/src/types/execution/block_body.rs

```diff
@@ -138,6 +138,7 @@ impl BlockBody {
         }
 
         trie.root_hash()
+            .map(|root| H256::from_slice(root.as_bytes()))
```

Comment:
Both are ethereum_types::H256, can't we skip the mapping?

Reply:
As discussed, it was needed before `eth_trie` was upgraded and new version published. No longer needed.